sql/moz-fx-data-shared-prod/firefox_accounts/fxa_all_events/view.sql

```diff
@@ -10,8 +10,10 @@ WITH fxa_auth_events AS (
   SELECT
     `timestamp`,
     receiveTimestamp,
+    TIMESTAMP_MILLIS(CAST(jsonPayload.fields.time AS INT64)) AS time,
```

Comment:
```suggestion
    TIMESTAMP_MILLIS(CAST(jsonPayload.fields.`time` AS INT64)) AS `time`,
```

Is `time` the best name for the field here? Would `timestamp_millisecond` or `millisecond_timestamp` be more informational?

Reply:
ok, just noticed the comment below related to this (renaming this field).